### PR TITLE
dropdown corner css issue fix

### PR DESCRIPTION
**Pull request by pdkpavankumar**
Pull Request for Issue # https://github.com/joomla/joomla-cms/issues/30117

### Summary of Changes
Fixed the corner css for the list of items present inside the dropdown

### Testing Instructions
any dropdown options
ex: article actions

### Actual result BEFORE applying this Pull Request

image

### Expected result AFTER applying this Pull Request

image

### Documentation Changes Required

No

### CODE REVIEW

File: libraries/src/Toolbar/Toolbar.php [hunk 1]
```diff
@@ -298,21 +298,27 @@ public function render(array $options = [])
 			$html[] = $layout->render(['id' => $this->_name]);
 		}
 
+		$i = 0;
+		$len = count($this->_bar);
+
 		// Render each button in the toolbar.
 		foreach ($this->_bar as $button)
```
Comment (Quy) [inline libraries/src/Toolbar/Toolbar.php]:
```suggestion
		$len = count($this->_bar);

		// Render each button in the toolbar.
		foreach ($this->_bar as $i => $button)
```

File: libraries/src/Toolbar/Toolbar.php [hunk 2]
```diff
@@ -298,21 +298,27 @@ public function render(array $options = [])
...
 				$html[] = $button->render();
 			}
 			// B/C
 			else
 			{
 				$html[] = $this->renderButton($button);
 			}
+
+			$i++;
 		}
```
Comment (Quy) [inline libraries/src/Toolbar/Toolbar.php]:
```suggestion
			}
		}
```

**Comment by pdkpavankumar** [comment]
@Quy Thanks for the suggestions

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 7f8d83c26d60c86ea4641aa65c75114e106543cb<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30596">issues.joomla.org/tracker/joomla-cms/30596</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 7f8d83c26d60c86ea4641aa65c75114e106543cb<br /><br />perfect - thanks very much<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30596">issues.joomla.org/tracker/joomla-cms/30596</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30596">issues.joomla.org/tracker/joomla-cms/30596</a>.</sub>

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**